<launch>
  <group ns="goal_hole">
    <!-- <rosparam param="measurement"> -->
    <!--   [0.5, -0.02, 0.3, 0.5, -0.02, 0.1] -->
    <!-- </rosparam> -->
    <!-- <rosparam param="measurement"> -->
    <!--   [0.32, -0.3, 0.3, 0.32, -0.3, 0.1] -->
    <!-- </rosparam> -->
    <rosparam param="measurement">
      <!-- [1.3, -0.04, 0.1, 1.1, -0.04, 0.1] -->
      [0.8, -0.02, -0.1, 0.8, -0.02, 0.1]
    </rosparam>


    <node pkg="touch_optimization" type="get_IG" 
	  name="simulating_measurement" output="screen"/>

  </group>

</launch>
